validator/accounts/wallet/wallet.go

```
@@ -290,6 +293,26 @@ func (w *Wallet) InitializeKeymanager(ctx context.Context, cfg iface.InitKeymana
 		if err != nil {
 			return nil, errors.Wrap(err, "could not initialize remote keymanager")
 		}
+	case keymanager.Web3Signer: //option should not work for --web
+		configFile, err := w.ReadKeymanagerConfigFromDisk(ctx)
+		if err != nil {
+			return nil, errors.Wrap(err, "could not read web3signer keymanager config")
+		}
+		config, err := remote_web3signer.UnmarshalConfigFile(configFile)
+		if err != nil {
+			return nil, errors.Wrap(err, "could not unmarshal web3signer keymanager config file")
+		}
+		// TODO(9883): future work needs to address how initialize keymanager is called for web3signer.
+		// an error may be thrown for genesis validators root for some InitializeKeymanager calls.
+		if len(cfg.GenesisValidatorsRoot) != fieldparams.RootLength {
+			config.GenesisValidatorsRoot = cfg.GenesisValidatorsRoot
+		} else {
+			return nil, errors.New("could not set genesis validators root for web3signer keymanager")
+		}
```

This should just throw an error like this:
```go
if len(cfg.GenesisValidatorsRoot) != 32 {
  return nil, errors.New("web3signer requires a genesis validators root value")
}
```